logstash-core-plugin-api/logstash-core-plugin-api.gemspec

```diff
@@ -0,0 +1,27 @@
+# -*- encoding: utf-8 -*-
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "logstash-core-plugin-api/version"
+# This is completely valid, when we do a `gem build` the gemspec is 
+# converted to a text file without any ruby code.
+require_relative "../logstash-core/lib/logstash-core/version"
+
+Gem::Specification.new do |gem|
+  gem.authors       = ["Elasticsearch"]
+  gem.email         = ["info@elasticsearch.com"]
+  gem.description   = %q{Logstash plugin api contract}
+  gem.summary       = %q{Define the plugin contract that the plugin need to follow.}
+  gem.homepage      = "http://www.elastic.co/guide/en/logstash/current/index.html"
+  gem.license       = "Apache License (2.0)"
+
+  gem.files         = Dir.glob(["logstash-core-event.gemspec", "lib/**/*.rb", "spec/**/*.rb"])
+  gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
+  gem.name          = "logstash-core-plugin-api"
+  gem.require_paths = ["lib"]
+  gem.version       = LOGSTASH_CORE_PLUGIN_API
+
+  gem.add_runtime_dependency "logstash-core", "#{LOGSTASH_CORE_VERSION}"
```

Comment:
right, my comment is more about the automatic insertion of that version instead of letting us setting it when we generate. I am thinking there might be cases where we'd want to generate a new plugin-api gem using a dependency on logstash-core that is not the one currently in the source tree.